### PR TITLE
[DOCFIX] Fix some errors occurred when formatting alluxio

### PR DESCRIPTION
**Pull request by jianghuazhu**
### What changes are proposed in this pull request?
Some documentation has been modified to suggest better use of alluxio.

### Why are the changes needed?
If JAVA_HOME is not set properly, you will encounter some errors when formatting.
Details: https://github.com/Alluxio/alluxio/issues/16949

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  1. change in user-facing APIs
  2. addition or removal of property keys
  3. webui

### CODE REVIEW

**Comment by jianghuazhu** [comment]
After adding JAVA_HOME configuration, reformat alluxio. We found that everything works fine, here are some logs:
[2023-02-27 17:00:07][localhost] 2023-02-27 17:00:07,486 INFO  Format - Formatting worker data folder: /alluxioworker/
[2023-02-27 17:00:07][localhost] 2023-02-27 17:00:07,486 WARN  InstancedConfiguration - PropertyKey alluxio.worker.tieredstore.level0.dirs.path's type is LIST, please use proper getter method for the type, getString will no longer work for non-STRING property types in 3.0
[2023-02-27 17:00:07][localhost] 2023-02-27 17:00:07,486 INFO  Format - Formatting Data path for tier 0:/mnt/ramdisk/alluxioworker
[2023-02-27 17:00:07][localhost] 2023-02-27 17:00:07,501 INFO  Format - Formatting complete
[2023-02-27 17:00:07][localhost] Connection to localhost closed.

**Comment by jianghuazhu** [comment]
@fuzhengjia @dbw9580, could you please help review this pr?
Thanks.

**Review by maobaolong — COMMENTED** [review]
@jianghuazhu If we define the JAVA_HOME in our os env, is this necessary to define it in `conf/alluxio-env.sh` again?

If not, you can add some condition description to explain what case to add `JAVA_HOME` manually.

**Comment by jianghuazhu** [comment]
@maobaolong, thank you so much for your comments.
I set 'JAVA_HOME' in the /etc/profile file and still get these errors. Therefore, I think configuring JAVA_HOME in the alluxi-env.sh file should be considered in advance.

File: docs/cn/overview/Getting-Started.md [hunk 1]
```diff
@@ -48,6 +48,18 @@ $ cd alluxio-{{site.ALLUXIO_VERSION_STRING}}
...
+在`${ALLUXIO_HOME}/conf`目录下，根据模板文件创建`conf/alluxio-env.sh`配置文件。
+
+```console
+$ cp conf/alluxio-env.sh.template conf/alluxio-env.sh
+```
+
+在`conf/alluxio-env.sh`文件中为 `JAVA_HOME` 增加配置.例如:
+
+```console
+$ echo "JAVA_HOME=/usr/java/jdk1.8.0_171" >> conf/alluxio-env.sh
```
Comment (dbw9580) [inline docs/cn/overview/Getting-Started.md:60]:
```suggestion
$ echo "JAVA_HOME=/path/to/java/home" >> conf/alluxio-env.sh
```

File: docs/en/overview/Getting-Started.md [hunk 2]
```diff
@@ -50,6 +50,19 @@ to as `${ALLUXIO_HOME}`.
...
+file by copying the template file.
+
+```console
+$ cp conf/alluxio-env.sh.template conf/alluxio-env.sh
+```
+
+In `conf/alluxio-env.sh`, adds configuration for `JAVA_HOME`. For example:
+
+```console
+$ echo "JAVA_HOME=/usr/java/jdk1.8.0_171" >> conf/alluxio-env.sh
```
Comment (dbw9580) [inline docs/en/overview/Getting-Started.md:63]: also here

**Review by dbw9580 — CHANGES_REQUESTED** [review]
Thanks for the improvement. Only a minor comment, otherwise LGTM.

**Review by JiamingMai — COMMENTED** [review]
We should not use a specific jdk path in the docs.

File: docs/cn/overview/Getting-Started.md (hunk 1 above)
Comment (jianghuazhu) [reply docs/cn/overview/Getting-Started.md:60]: Thanks.
I will update soon.

**Comment by jianghuazhu** [comment]
@dbw9580  @JiamingMai , I have updated, can you review this pr again?
thanks.

**Comment by maobaolong** [comment]
@dbw9580 Did you met this issue? It run good without `JAVA_HOME` defined in our  `conf/alluxio-env.sh`

**Comment by jianghuazhu** [comment]
@maobaolong, thanks for your comments.
Here is the environment in which I ran alluxio:
1. Operating system :Ubuntu 14.04LTS
2. In the /etc/profile file, set the JAVA_HOME and alluxio directories.
export JAVA_HOME=/usr/java/jdk1.8.0_171
export ALLUXIO_HOME = / home/jianghuazhu/alluxio/deplay/alluxio - 2.9.1
3. Check the java environment, java -version. The following information is displayed:
java version "1.8.0_171"
Java(TM) SE Runtime Environment (build 1.8.0_171-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.171-b11, mixed mode)

**Comment by jianghuazhu** [comment]
@maobaolong @dbw9580 @JiamingMai, can you review this pr again?

**Comment by elega** [comment]
This looks good to me @JiamingMai @dbw9580 can you guys take a second look?

**Review by JiamingMai — APPROVED** [review]
LGTM

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**